xDripG5/CalibrationState.swift

```diff
@@ -93,3 +93,14 @@ extension CalibrationState: Equatable {
         }
     }
 }
+
+extension CalibrationState: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .known(let state):
+            return "\(state)"
+        default:
```

Comment:
This should be the `.unknown` case; default cases should be avoided whenever possible.